doc/tutorials/gapi/oak_devices/oak_devices.markdown

```diff
@@ -0,0 +1,26 @@
+Using DepthAI Hardware /OAK depth sensors {#tutorial_gapi_oak_devices}
+=======================================================================================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_gapi_face_beautification}
+
+![hardwares](pics/oak.jpg)
+
+Depth sensors compatible with Luxonis DepthAI library are supported through OpenCV Graph API (or G-API) module. Depth map, RGB image and some other formats of output can be retrieved by using familiar interface of G-API module.
```

Comment:
I don't think `depth map` can be retrieved via G-API currently. It can retrieve RGB and NV12 formats though

Reply:
ok

Reply:
Done master @smirnov-alexey  and @dmatveev .  I'm studying GAPI with OAK to detail in the next request.